README.md

```diff
@@ -1,6 +1,29 @@
-# Memory-game
+Memory Game
```

Comment:
Excellent add of the README with instructions and details, it's important and very useful for anyone who wants to try your app. Well done! 🚀

💡 You can format the README using Markdown or even add images or videos: [GitHub Markdown](https://docs.github.com/es/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

Reply:
I'll work on it :)